package.json

```diff
@@ -44,7 +44,33 @@
 				"label": "File Icons (Colourless)",
 				"path": "./icons/file-icons-colourless-icon-theme.json"
 			}
-		]
+		],
+		"commands": [
+			{
+				"command": "file-icons-colourless.hidesExplorerArrows",
+				"title": "%command.hidesExplorerArrows%"
```

Comment:
I tested this locally, and received the following error:

<img src="https://user-images.githubusercontent.com/2346707/159123828-fc47d6f4-ec25-4f4d-a8a5-d7b58a69c378.png" alt="Command '%command.hidesExplorerArrows%' resulted in an error (command 'file-icons.hidesExplorerArrows' not found)" width="288" />

Reply:
Let me try to replicate that error